src/Microsoft.AspNetCore.Mvc.RazorPages/Internal/PageBinderFactory.cs

```diff
@@ -0,0 +1,174 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc.Internal;
+using Microsoft.AspNetCore.Mvc.ModelBinding;
+using Microsoft.AspNetCore.Mvc.RazorPages.Infrastructure;
+
+namespace Microsoft.AspNetCore.Mvc.RazorPages.Internal
```

Comment:
Can we please stop this? This has no reason to be user-visible and extensible

Reply:
I renamed this type without changing any of it's modifiers, git just shows it as a new type. But are you suggesting we do away with pub-internal types?